opentech/apply/determinations/templates/determinations/includes/determination_button.html

```diff
@@ -0,0 +1,12 @@
+{% load determination_tags workflow_tags %}
+{% if request.user|is_lead_or_admin:submission %}
+    {% if request.user|can_add_determination:submission %}
```

Comment:
this looks like we are performing two checks which do the same thing. I dont kknow the difference form looking at it.

Reply:
simplified.